In [12]:
from scipy.optimize import minimize

fun = lambda x: (x[0]-x[1])**2 + (x[1]+x[2]-2)**2 + (x[3]-1)**2 + (x[4]-1)**2
con = ({'type': 'eq', 'fun': lambda x: x[0]+3*x[1]},
        {'type': 'eq', 'fun': lambda x: x[2]+x[3]-2*x[4]},
        {'type': 'eq', 'fun': lambda x: x[1]-x[4]})
bnd = ((-10,10),(-10,10),(-10,10),(-10,10),(-10,10))
res = minimize(fun, (100,200,100,500,800), method = 'SLSQP', bounds = bnd, constraints = con)

In [8]:
res.fun

4.09302326452976

In [9]:
res.fun

4.09302326452976

In [11]:
res.fun

4.093023258012402

The first run, my initial guess was (1,1,1,1,1).
The second run, my initual guess was (1,2,3,4,5).
The third run, my initial guess was (60,80,50,60,40).

From these guesses, I can tell that there isn't a drastic change in the solution. The percent differnce between these solutions is negligible and if my initital guess is anything less than 100, the solution will be accurate to 7 significant figures. 

In [13]:
res.fun

4.093023258012402

Another observation is when the initial guesses are small, the solution approaches 4.09302326452976, but when the initial guesses are large, the solution approaches 4.093023258012402.